src/Cupboard.Providers.Windows/Certificate/Certificate.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Security.Cryptography.X509Certificates;
+using Spectre.IO;
+
+namespace Cupboard
+{
+    public sealed class Certificate : Resource
+    {
+        public Uri? Url { get; set; }
+
+        public FilePath? FilePath { get; set; }
+
+        public StoreLocation StoreLocation { get; set; }
+
+        public StoreName StoreName { get; set; }
+
+        public CertificateState Ensure { get; set; }
+
+        public string Thumbprint { get; set; } = string.Empty;
```

Comment:
Not sure if I understand completely, but shouldn't this be nullable?

Reply:
Properly :)